### ISSUE TITLE
Typing - Missing accessibilityLabel in ReactVideoProps

### ISSUE THREAD

**Issue by delphinebugner** [TheWidlarzGroup/react-native-video#3427]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-video@6.0.0-beta.2` for the project I'm working on.

After the upgrade (from 5.2.1 to 6.0.0-beta.2), typing broke for accessibilityLabel of Video - even if it is actually taken into account in the test after!

### Without the fix:
`TS error= Property 'accessibilityLabel' does not exist on type 'IntrinsicAttributes & ReactVideoProps'
`
```typescript
import Video, { IgnoreSilentSwitchType, ReactVideoProps, ResizeMode } from 'react-native-video';

export const TF1Video = (props: ReactVideoProps) => {
  return (
    <Video
      muted={true}
      repeat={true}
      resizeMode={ResizeMode.COVER}
      rate={1.0}
      ignoreSilentSwitch={IgnoreSilentSwitchType.OBEY}
      accessibilityLabel="TF1video" // 🐞🐞  TS error= Property 'accessibilityLabel' does not exist on type 'IntrinsicAttributes & ReactVideoProps'
      {...props}
    />
  );
};

``` 

It was okay in previous version of lib (5.2.1). 
It's only a TS issue, in my test `screen.getByLabelText('TF1video')` works! 👌 

### With the fix 
After adding it: TypeScript happy 😊 
<img width="681" alt="image" src="https://github.com/react-native-video/react-native-video/assets/67843879/afac6e8c-211c-4e6a-8a9b-8b40f4b97014">
 

### Here is the diff that solved my problem:

```diff
diff --git a/node_modules/react-native-video/lib/types/video.d.ts b/node_modules/react-native-video/lib/types/video.d.ts
index f61eda2..96b43d1 100644
--- a/node_modules/react-native-video/lib/types/video.d.ts
+++ b/node_modules/react-native-video/lib/types/video.d.ts
@@ -129,6 +129,7 @@ export declare enum PosterResizeModeType {
     STRETCH = "stretch"
 }
 export interface ReactVideoProps extends ReactVideoEvents {
+    accessibilityLabel?: string;
     source?: ReactVideoSource;
     drm?: Drm;
     style?: StyleProp<ViewStyle>;

```

On the lib, it could / should probably be on the `types/video.ts` file? 
I may open a PR for it, if pertinent!

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by KrzysztofMoch** [comment]
Hey 👋, @delphinebugner
How about make `ReactVideoProps` extends `AccessibilityProps` ? It should do the job
I would appreciate it if you could make a PR!

**Comment by freeboub** [comment]
@KrzysztofMoch @delphinebugner would it make sens to also add testID prop ?!

**Comment by delphinebugner** [comment]
Great idea for the accessibility props, it will add everything @KrzysztofMoch! I'll do it soon 🙂

@freeboub I think accessibility label is a better standard than test ID, but I guess the last ones are still widely used, so yes we should add them to!

**Comment by KrzysztofMoch** [comment]
Will be available in next release (6.0.0-beta.3). 
@delphinebugner Thank you for you PR 🚀
